media-gfx/blender/blender-4.4.3-r1.ebuild

```
@@ -0,0 +1,900 @@
+
+# NOTE BLENDER_VERSION
+# https://projects.blender.org/blender/blender/src/branch/main/source/blender/blenkernel/BKE_blender_version.h
+BLENDER_BRANCH="$(ver_cut 1-2)"
+
+if [[ "${PV}" == *9999* ]]; then
+	EGIT_LFS="yes"
+	inherit git-r3
+	EGIT_REPO_URI="https://projects.blender.org/blender/blender.git"
+	EGIT_SUBMODULES=( '*' '-lib/*' )
+
+	if [[ "${PV}" == 9999* ]]; then
+		EGIT_BRANCH="main"
+	else
+		EGIT_BRANCH="blender-v${BLENDER_BRANCH}-release"
+	fi
+
+else
+	SRC_URI="
+		https://download.blender.org/source/${P}.tar.xz
+		test? (
+			https://download.blender.org/source/blender-test-data-${BLENDER_BRANCH}.0.tar.xz
+		)
+	"
+	KEYWORDS="~amd64 ~arm ~arm64"
+fi
+
+# assets is CC0-1.0
+LICENSE="GPL-3+ cycles? ( Apache-2.0 ) CC0-1.0"
+SLOT="${BLENDER_BRANCH}"
+
+# NOTE +openpgl breaks on very old amd64 hardware
+# potentially mirror cpu_flags_x86 + REQUIRED_USE
+IUSE="
+	alembic +bullet collada +color-management cuda +cycles +cycles-bin-kernels
+	debug doc +embree +ffmpeg +fftw +fluid +gmp gnome hip hiprt jack
+	jemalloc jpeg2k man +nanovdb ndof nls +oidn openal +openexr +opengl +openmp +openpgl
+	+opensubdiv +openvdb optix osl pipewire +pdf +potrace +pugixml pulseaudio
+	renderdoc sdl +sndfile +tbb test +tiff +truetype valgrind vulkan wayland +webp X
+"
+
+if [[ "${PV}" == *9999* ]]; then
+	IUSE+="experimental"
+fi
+
+RESTRICT="!test? ( test )"
+
+REQUIRED_USE="${PYTHON_REQUIRED_USE}
+	|| ( opengl vulkan )
+	alembic? ( openexr )
+	cuda? ( cycles )
+	cycles? ( openexr tiff tbb )
+	fluid? ( tbb )
+	gnome? ( wayland )
+	hip? ( cycles )
+	hiprt? ( hip )
+	nanovdb? ( openvdb )
+	openvdb? ( tbb openexr )
+	optix? ( cuda )
+	osl? ( cycles pugixml )
+	test? (
+		color-management
+		jpeg2k
+	)
+"
+
+# Library versions for official builds can be found in the blender source directory in:
+# build_files/build_environment/cmake/versions.cmake
+RDEPEND="${PYTHON_DEPS}
+	app-arch/zstd
+	dev-cpp/gflags:=
+	dev-cpp/glog:=
+	dev-libs/boost:=[nls?]
+	dev-libs/lzo:2=
+	$(python_gen_cond_dep '
+		dev-python/cython[${PYTHON_USEDEP}]
+		dev-python/numpy[${PYTHON_USEDEP}]
+		dev-python/zstandard[${PYTHON_USEDEP}]
+		dev-python/requests[${PYTHON_USEDEP}]
+	')
+	media-libs/freetype:=[brotli]
+	media-libs/libepoxy:=
+	media-libs/libjpeg-turbo:=
+	media-libs/libpng:=
+	media-libs/libsamplerate
+	>=media-libs/openimageio-2.5.6.0:=
+	sys-libs/zlib:=
+	virtual/glu
+	virtual/libintl
+	virtual/opengl[X?]
+	alembic? ( >=media-gfx/alembic-1.8.3-r2[boost(+),hdf(+)] )
+	bullet? ( sci-physics/bullet:=[double-precision] )
+	collada? ( >=media-libs/opencollada-1.6.68 )
+	color-management? ( media-libs/opencolorio:= )
+	cuda? ( dev-util/nvidia-cuda-toolkit:= )
+	embree? ( media-libs/embree:=[raymask] )
+	ffmpeg? ( media-video/ffmpeg:=[encode(+),lame(-),jpeg2k?,opus,theora,vorbis,vpx,x264,xvid] )
+	fftw? ( sci-libs/fftw:3.0=[threads] )
+	gmp? ( dev-libs/gmp[cxx] )
+	gnome? ( gui-libs/libdecor )
+	hip? (
+		>=dev-util/hip-5.7:=
+	)
+	hiprt? ( media-libs/hiprt:2.5= )
```

```suggestion
	hip? (
		>=dev-util/hip-5.7:=
		hiprt? (
			dev-libs/hiprt:2.5=
		)
	)
```

https://projects.blender.org/blender/blender/src/tag/v4.4.3/intern/cycles/cmake/external_libs.cmake#L66-L68
Can't have `hiprt` without `hip`, so we can simplify the depspec.